### ISSUE TITLE
//UPLOAD not working on Linux

### ISSUE THREAD

**Issue by the-godfather-007** [wwivbbs/wwiv#1631]
//Upload to upload a file for the purpose of posting is not working, as a result uploading ASCII or ANSI files to post on the WWIVnet ANSI subboard is not achievable at present.  The menu command exists, however when typing //upload from main just refreshes the main menu.

**Comment by the-godfather-007** [comment]
one observation with Linux and WWIV or possibly not OS specific but rather just a //upload issue (or //uploadall also).  I've noticed that WWIV only seems to like .zip files and the files need to be UPPER case with linux.  I have also found that if the hard drive is not immediately responsive it will show my files as offline.  I have 3 12tb HD's and an external 6tb HD, plus two 1TB m.2 SSD drives.  The 6tb external takes a moment to fire up if "asleep," and WWIV skips right past it.  WWIV seems to like my m.2 SSD the best  as that is the drive where /wwiv/dloads exists.  Anything outside of that structure such as /mnt/username/partisan4/afiledirectory/achilddirectory or any combination of alt mount points WWIV does not appear to like.  Lastly, as I belive I've stated before, //UPLOAD will SEE the files on the hard drive but will not save them to data files to be listed by users later (allow.dat?)  Lastly, when removing a file it asks "remove from allow.dat?"  When selecting YES, removing the file from the directory (or renaming it,) restarting WWIVD services, and rerunning //upload, the original file no longer on the HD reappears as an "added" or "uploaded file."  This may require a time one logs onto my system, I give temp sysop, and then show live.  But just thought I would add context as I can't seem to pinpoint the cause for WWIV not fully loading all files within directories.  Lastly, I attempted to upload the WWIV FTP files and their directories from within my /wwiv/dloads and 90% were "ignored" and not added to the download list.  I don't know if this is a linux to dos or dos to linux file conversion, older or newer archivers  (pkzip, 7z, .zip, tar, etc..) or what.  I can't figure it out.  This is beyond my scope.
